.github/workflows/test-old-node-error.yml

```diff
@@ -0,0 +1,24 @@
+name: Test old node.js version
+
+on: pull_request
+
+jobs:
+  check:
+    name: "Checks"
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
How about a matrix of older versions of node that we would like to check? E.g. 10, 12, 14?

Reply:
considered it, didn't want to over do it tbh